examples/kernel_approximation/plot_scalable_poly_kernels.py

```diff
@@ -99,7 +99,7 @@
 # compensate for the stochastic nature of :class:`PolynomialCountSketch`.
 
 n_runs = 3
```

Comment:
I am thinking that we could maybe avoid several runs and just mention that one should repeat it to have a more stable estimate in practice. I think that this is the only way that we can speed up, even more, the experiment.
@ogrisel WDYT?

Reply:
@glemaitre Alright, sounds good :)

Reply:
I agree, it sounds unnecessary to me to run this 3 times. We should either do it at least 5 times, or only once, and just say it can be run a few times to get a more reliable result there.